src/stactools/modis/utils.py

```diff
@@ -31,11 +42,62 @@ def version_string(version: str) -> str:
         raise ValueError(f"Unsupported MODIS version: {version}")
 
 
-def recursive_round(coordinates: List[Any], precision: int = 6) -> List[Any]:
+def recursive_round(coordinates: List[Any]) -> List[Any]:
```

Comment:
```suggestion
def recursive_round(coordinates: List[Any]) -> List[Any]:
    # TODO: use stactools.core.utils.round.recursive_round once it is released in v0.4.4 or v0.5
```

Just so future us remembers.

Reply:
Now using recursive_round in stactools v0.4.4